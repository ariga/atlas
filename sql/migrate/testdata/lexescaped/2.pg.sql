create table t1 (c text default '\');
create table t1 (c text default E'\\');
create table t1 (c text default '\A\');
create table t1 (c text default e'\\A\\');