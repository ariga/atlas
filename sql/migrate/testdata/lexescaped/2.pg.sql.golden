create table t1 (c text default '\');
-- end --
create table t1 (c text default E'\\');
-- end --
create table t1 (c text default '\A\');
-- end --
create table t1 (c text default e'\\A\\');